<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingWindow</class>
 <widget class="QWidget" name="SettingWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>932</width>
    <height>531</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0" colspan="2">
    <widget class="QTabWidget" name="tabWidget">
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="programSetting">
      <attribute name="title">
       <string>Tab 1</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QCheckBox" name="boxAutoStart">
         <property name="text">
          <string>设置开机自启动</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="boxSlientBoot">
         <property name="text">
          <string>程序静默启动（启动后不自动弹出搜索栏）</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="boxPreLoadResource">
         <property name="text">
          <string>设置启动预加载资源（程序启动会慢一些）</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QLineEdit" name="LEPlaceholderText"/>
         </item>
         <item>
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>自定义搜索栏的提示文本</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <widget class="QLineEdit" name="LEEmptyText"/>
         </item>
         <item>
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>自定义搜索无结果的文本</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_7" stretch="1,1,8">
         <item>
          <widget class="QSpinBox" name="spResultItemNumber">
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>10</number>
           </property>
           <property name="value">
            <number>4</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>设置搜索结果数量</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_10" stretch="1,1,8">
         <item>
          <widget class="QSpinBox" name="spReloadTime">
           <property name="minimum">
            <number>5</number>
           </property>
           <property name="maximum">
            <number>9999</number>
           </property>
           <property name="value">
            <number>30</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>数据库刷新时间（分钟）</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QCheckBox" name="boxEnableStatistics">
         <property name="text">
          <string>启用程序打开次数统计（将会用于搜索算法中）</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="pathSetting">
      <attribute name="title">
       <string>Tab 2</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>自定义搜索路径</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QListWidget" name="searchList"/>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <widget class="QPushButton" name="btnResetSearchList">
              <property name="text">
               <string>重置</string>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout">
              <item>
               <widget class="QPushButton" name="btnAddSearchItem">
                <property name="text">
                 <string>添加一行</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="btnDelSearchItem">
                <property name="text">
                 <string>删除选中行</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <spacer name="verticalSpacer_3">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QCheckBox" name="boxSearchStartMenu">
              <property name="enabled">
               <bool>true</bool>
              </property>
              <property name="text">
               <string>搜索开始目录文件夹</string>
              </property>
              <property name="checkable">
               <bool>true</bool>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="boxSearchProgramFile">
              <property name="text">
               <string>搜索Program File文件夹</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="boxSearchUWP">
              <property name="text">
               <string>搜索UWP应用</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="boxSearchRegistry">
              <property name="text">
               <string>通过注册表查找</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string>自定义屏蔽路径</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QListWidget" name="bannedList"/>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <widget class="QPushButton" name="btnResetBannedList">
              <property name="text">
               <string>重置</string>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_3">
              <item>
               <widget class="QPushButton" name="btnAddBannedItem">
                <property name="text">
                 <string>添加一行</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="btnDelBannedItem">
                <property name="text">
                 <string>删除选中行</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <spacer name="verticalSpacer">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Page</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_8">
       <item>
        <widget class="QTableView" name="tableKeyFilter"/>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <widget class="QPushButton" name="btnResetKeyTable">
           <property name="text">
            <string>重置</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnAddKeyItem">
           <property name="text">
            <string>添加一行</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnDelKeyItem">
           <property name="text">
            <string>删除选中行</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnIgnoreUnstallApp">
           <property name="text">
            <string>忽略卸载程序</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnIgnoreHelpApp">
           <property name="text">
            <string>忽略帮助文档</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer_4">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Page</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_9">
       <item>
        <widget class="QTableWidget" name="tableIndexedApp"/>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <item>
          <widget class="QPushButton" name="pushButton">
           <property name="text">
            <string>刷新</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer_5">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QPushButton" name="btnConfirm">
     <property name="text">
      <string>确定</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLineEdit" name="lineEdit">
     <property name="text">
      <string>软件更新地址：https://github.com/ghost-him/ZeroLaunch-CPP/releases</string>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
